app/src/lib/design-system-v2/helpers/theme.css

```diff
@@ -80,5 +80,32 @@
   --requestly-color-white: #ffffff;
   --requestly-color-white-t-20: #ffffff33;
   --requestly-color-black: #0e0e0e;
-  --requestly-color-black-t-70: 000000b2;
+  --requestly-color-black-t-70: #000000b2;
+  --requestly-font-weight-thin: 100;
+  --requestly-font-weight-extralight: 200;
+  --requestly-font-weight-light: 300;
+  --requestly-font-weight-normal: 400;
+  --requestly-font-weight-medium: 500;
+  --requestly-font-weight-semibold: 600;
+  --requestly-font-weight-bold: 700;
+  --requestly-font-weight-extrabold: 800;
+  --requestly-font-weight-black: 900;
+  --requestly-font-size-xs: 0.75rem;
+  --requestly-font-size-sm: 0.875rem;
+  --requestly-font-size-base: 1rem;
+  --requestly-font-size-lg: 1.125rem;
+  --requestly-font-size-xl: 1.25rem;
+  --requestly-font-size-2xl: 1.5rem;
+  --requestly-font-size-3xl: 1.875rem;
+  --requestly-font-size-4xl: 2.25rem;
+  --requestly-font-size-5xl: 3rem;
+  --requestly-font-size-6xl: 3.75rem;
+  --requestly-font-line-height-xs: 1rem;
+  --requestly-font-line-height-sm: 1.25rem;
+  --requestly-font-line-height-base: 1.5rem;
+  --requestly-font-line-height-lg: 1.75rem;
+  --requestly-font-line-height-xl: 1.75rem;
+  --requestly-font-line-height-2xl: 2rem;
+  --requestly-font-line-height-3xl: 2.25rem;
+  --requestly-font-line-height-4xl: 2.5rem;
```

Comment:
We should add a doc comment for there pixel equivalent values e.g
```css
/* 12px */
 --requestly-font-size-xs: 0.75rem;
 ```

Reply:
theme.css is just for reference. In future we'll make this dynamic depending upon user selected font size

Reply:
We'll not need the px value as all the values defined in designs will be using these values only